docs/sphinx/install.rst

```diff
@@ -106,69 +110,10 @@ or run the Python examples using the Python interpreter.
 Build CUDA Quantum from Source
 ------------------------------
 
-Here we will assume a Ubuntu 22.04 system. Adjust the package manager calls
-for your distribution. Make sure that recent versions `cmake` and `ninja` installed.
-The build also requires a recent version of `clang/clang++` or `gcc/g++`
-(must have C++20 support).
-
-Get the basic compilers you'll need via apt-get
-+++++++++++++++++++++++++++++++++++++++++++++++
-.. code:: bash
-  
-    apt-get update && apt-get install -y --no-install-recommends gcc g++ 
-
-On Ubuntu 22.04 this will get you GCC 11. 
-
-Get cuQuantum (optional)
-++++++++++++++++++++++++
-
-.. code:: bash 
-    
-    wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb \
-    dpkg -i cuda-keyring_1.0-1_all.deb
-    apt-get update && apt-get -y install cuquantum cuquantum-dev 
-
-Get LLVM / Clang / MLIR
-++++++++++++++++++++++++
-
-You will need the same version of LLVM as our submodule in `tpls/llvm`.
-
-.. code:: bash 
-
-    mkdir llvm-project && cd llvm-project
-    git init 
-    git remote add origin https://github.com/llvm/llvm-project 
-    # note this will change as the project evolves, 
-    # Must be == to the hash we use for the tpls/llvm submodule.
-    git fetch origin --depth=1 c0b45fef155fbe3f17f9a6f99074682c69545488
-    git reset --hard FETCH_HEAD
-    mkdir build && cd build
-    cmake .. -G Ninja  
-                -DLLVM_TARGETS_TO_BUILD="host" \
-                -DCMAKE_INSTALL_PREFIX=/opt/llvm/
-                -DLLVM_ENABLE_PROJECTS="clang;mlir" 
-                -DCMAKE_BUILD_TYPE=Release 
-                -DLLVM_ENABLE_ASSERTIONS=ON 
-                -DLLVM_INSTALL_UTILS=TRUE 
-    ninja install
-    # This is needed for FileCheck tests.
-    cp bin/llvm-lit /opt/llvm/bin/
-
-Build CUDA Quantum
-++++++++++++++++++
-You must use the same compiler that you compiled LLVM with to compile CUDA Quantum.
-
-.. code:: bash
-    
-    git clone https://github.com/NVIDIA/cuda-quantum && cd cuda-quantum
-    mkdir build && cd build
-    cmake .. -G -DCMAKE_INSTALL_PREFIX=$HOME/.cudaq 
-                -DLLVM_DIR=/path/to/llvm/lib/cmake/llvm 
-                -DCUDAQ_ENABLE_PYTHON=TRUE
-                \# (optional, if cuquantum is installed)
-                -DCUSTATEVEC=/opt/nvidia/cuquantum
-    ninja install
-    ctest 
+For more information about building CUDA Quantum from source, 
+we refer to the `CUDA Quantum GitHub repository`_.
+
+.. _CUDA Quantum GitHub repository: https://github.com/NVIDIA/cuda-quantum
```

Comment:
Needs to add these instructions to the repository if we are telling users to go looking for them here.

Reply:
They are in the repository.

Reply:
My bad, I missed the `Building.md` in the repo.